docs/HowTo/Get-Started/Register-Validators.md

```diff
@@ -0,0 +1,129 @@
+
+```bash
+besu --genesis-file=depositContractGenesis.json \
+--data-path=data --miner-enabled=true \
+--miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 \
+--rpc-http-cors-origins="all" --host-whitelist="*" \
+--min-gas-price=0 --rpc-http-enabled=true \
+--rpc-http-port=8545 \
+--rpc-http-api=ETH,IBFT,CLIQUE,DEBUG,MINER,NET,PERM,ADMIN,EEA,TXPOOL,PRIV,WEB3
+``` 
+
+## Submit deposits
+
+You can register validators and submit the required deposits from the command line.
+
+The deposit amount required to register a validator is funded by an Ethereum 1.0 account
+specified using the [`--eth1-private-key`](../../Reference/CLI/CLI-Subcommands.md#eth1-private-key)
+or [`--eth1-keystore-file`](../../Reference/CLI/CLI-Subcommands.md#eth1-keystore-file) option.
+
+### Generate new validators keys
+
+Generate the keys for the validators and send deposits to the deposit contract.
+
+```bash
+teku validator generate --deposit-amount-gwei=32000000000 \
+--eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd \
+--eth1-endpoint=http://localhost:8545 --encrypted-keystore-enabled=false \
+--keys-output-path=validator_keys \
+--eth1-private-key=8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63 \
+--number-of-validators=64
+```
+
+The command line:
+
+* Specifies the amount of ether to deposit in Gwei using
+[`--deposit-amount-gwei`](../../Reference/CLI/CLI-Subcommands.md#deposit-amount-gwei) 
+* Specifies the address of the deposit contract using 
+[`--eth1-deposit-contract-address`](../../Reference/CLI/CLI-Subcommands.md#eth1-deposit-contract-address)
+* Specifies the JSON-RPC URL endpoint of the Ethereum 1.0 client (Besu) using
+[`--eth1-endpoint`](../../Reference/CLI/CLI-Subcommands.md#eth1-endpoint)
+* Enables encrypting the validator keys using 
+[`--encrypted-keystore-enabled`](../../Reference/CLI/CLI-Subcommands.md#encrypted-keystore-enabled)
+* Specifies the output location of the validator keys using
+[`--keys-output-path`](../../Reference/CLI/CLI-Subcommands.md#keys-output-path)
+* Specifies the private key that sends the transaction to the deposit contract using
+[`--eth1-private-key`](../../Reference/CLI/CLI-Subcommands.md#eth1-private-key)
+* Specify the number of validator keys and deposits to create using
+[`--number-of-validators`](../../Reference/CLI/CLI-Subcommands.md#number-of-validators).
+
+### Register an existing validator key
+
+You can register an existing validator key and send deposits to the deposit contract.
+
+```bash
+teku validator register --deposit-amount-gwei=32000000000 \
+--encrypted-keystore-validator-file=validator_0xa245bd5162e944efac2546bcadbba5fa2c8929ec6b03380d46fb0e8a6f49b176fecaf1939ed31532711375f873da58cf.json \
+--encrypted-keystore-validator-password-file=password.txt \
+--eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd \
+--eth1-endpoint=http://localhost:8545 \
+--eth1-private-key=8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63 \
```

Comment:
We can also specify v3 encrypted private keystore file/password file for Eth1 private key, instead of specifying it on command line. Similar to encrypted validator keystore file/password.

Reply:
Done